src/main/java/duke/Duke.java

```diff
@@ -1,333 +1,54 @@
 package duke;
 
-/*
- * Import Java Packages
- */
-import java.util.Scanner;
-import java.util.ArrayList;
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.FileWriter;
-import java.io.IOException;
 
-/*
- * Import Duke Packages
- */
-import duke.command.DukeException;
-import duke.task.Deadline;
-import duke.task.Event;
-import duke.task.Task;
-import duke.task.Todo;
+import duke.ui.TextUi;
+import duke.parser.Parser;
+import duke.command.*;
```

Comment:
Do import classes explicitly, you may have to change IntelliJ settings.

https://se-education.org/guides/conventions/java/basic.html#package-and-import-statements